src/views/profile-settings/Family.vue

```diff
@@ -103,17 +104,21 @@ export default class AppMain extends AlertMixin {
 
   async mounted() {
     this.user = await Authentication.instance.getCurrentUser();
-
     this.family = await CurrentFamily.instance.getCurrentFamily();
-    // TODO: resolve CORS
-    // this.familyMembers = await Firestore.instance.getUsersByEmail(this.members);
 
     await CurrentFamily.instance.listenForChanges(snapshot => {
       const family = snapshot.data() as Family;
       this.members = family?.members ?? [];
       this.pendingMembers = family?.pendingMembers ?? [];
       this.family = family;
     });
+
+    try {
+      // TODO: resolve CORS
+      this.familyMembers = await Firestore.instance.getUsersByEmail(this.members);
```

Comment:
`this.members` will be empty on start.
```suggestion
      this.familyMembers = await Firestore.instance.getUsersByEmail(this.family.members);
```